src/MobileRedirection.php

```diff
@@ -44,6 +44,7 @@ public function register() {
 		add_filter( 'amp_default_options', [ $this, 'filter_default_options' ] );
 		add_action( 'amp_options_menu_items', [ $this, 'add_settings_field' ], 10 );
 		add_filter( 'amp_options_updating', [ $this, 'sanitize_options' ], 10, 2 );
+		// @todo Add noamp=mobile to amp-view link in admin bar.
```

Comment:
Is this still valid?

Reply:
Not entirely since redirection is not automatically performed when in Dev Mode (thus when the admin bar is shown) but I've added logic for this to be sure: df52e281dab4f6bbb66b3219d8bedcfed49f326d